### PR TITLE
Added functionality to add a user to the Firebase list

### PR DESCRIPTION
**Pull request by dpberry552**
The "Add New User" button will now pass the text from the new user input
box to the "addUser" method, which calls a service to add the new name
to the list of usernames stored in Firebase.

### CODE REVIEW

File: src/app/twitch-manage-users/twitch-manage-users.component.html [hunk 1]
```diff
@@ -19,15 +12,24 @@
...
+          <div class="col-sm-10">
+            <input type="text" class="form-control" id="inputNewUser" placeholder="Add new user..." [(ngModel)]="newUser">
+          </div>
+        </div>
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-        <button type="button" class="btn btn-primary">Save changes</button>
+        <button type="button" class="btn btn-danger">Delete Selected</button>
+        <button (click)="addNewUser(newUser)" type="button" class="btn btn-primary">Add New User</button>
```
Comment (box86rowh) [inline src/app/twitch-manage-users/twitch-manage-users.component.html]: you can just do - addNewUser() here

File: src/app/twitch-manage-users/twitch-manage-users.component.ts [hunk 2]
```diff
@@ -5,16 +5,25 @@ import { AngularFire, FirebaseListObservable } from 'angularfire2';
...
   constructor(private service: TwitchDatabaseService) { }
 
     userNames: FirebaseListObservable<any> = this.service.getUsers();
-
+    newUser: string;
   ngOnInit() {
     console.log(this.userNames);
   }
 
+  addNewUser(un: string){
```
Comment (box86rowh) [inline src/app/twitch-manage-users/twitch-manage-users.component.ts]: you can just do addNewUser() - no need to pass the username in, it is already in scope, you can just refer to this.userName within this function

**Review by box86rowh — CHANGES_REQUESTED** [review]
see comments for requested changes

**Comment by dpberry552** [comment]
Made the change...did you see it come through?

**Comment by box86rowh** [comment]
yes! Merging...

**Pull request merged**